src/App.js

```diff
@@ -33,9 +33,9 @@ function App() {
           <div className='container'>
             <Route exact path = '/' component={SplashPage}/>
             <Route exact path = '/login'component={LoginForm}/>
-            <Route exact path = '/stocks1'component={Stocks1}/>
```

Comment:
Please add Barnes to review this section

Reply:
sure

Reply:
sure